pkg/cnb/store_buildpack_repository.go

```diff
@@ -93,9 +94,14 @@ func (s *StoreBuildpackRepository) layersForOrder(order v1alpha1.Order) ([]build
 	return buildpackLayers, nil
 }
 
-func highestVersion(matchingBuildpacks []v1alpha1.StoreBuildpack) v1alpha1.StoreBuildpack {
+func highestVersion(matchingBuildpacks []v1alpha1.StoreBuildpack) (v1alpha1.StoreBuildpack, error) {
+	for _, bp := range matchingBuildpacks {
+		if _, err := semver.NewVersion(bp.Version); err != nil {
+			return v1alpha1.StoreBuildpack{}, errors.Errorf("cannot find buildpack '%s' with latest version due to invalid semver '%s'", bp.Id, bp.Version)
```

Comment:
Really smart. I like the verify it is valid Semver and then sort with `semver.MustParse`

Reply:
Makes this really simple.